### PR TITLE
Update

### PR DESCRIPTION
**Pull request by aintgoin2goa**
Hopefully this will allow us to use both ways of updating flags while api is in beta?

@matthew-andrews

### CODE REVIEW

**Comment by matthew-andrews** [comment]
what does this do? merge them together?

**Comment by aintgoin2goa** [comment]
It replaces the working copy with data from the API.  So if someone needed to still use this repo that would need to run `make update` **first** and then make their changes as normal.

I really want to have a beta period for the API to see if it actually works - but people might find it too annoying to create new flags that way so wanted to give them another option.

**Comment by matthew-andrews** [comment]
Starting Monday let's `@channel` everyone in `ft-next-dev` to encourage people to use flags api **only** (and probably fail the build if they try to use this repo) — but still great to have this code (so :+1: for this PR) as a secret backup if the new API doesn't work.

File: Makefile [hunk 1]
```diff
@@ -19,6 +19,9 @@ build-production: build
 deploy: build
 	node deploy.js prod
 
+update:
+	node update/update.js
```
Comment (matthew-andrews) [inline Makefile]: do we want to update the CI deployment of this to run `update` before `deploy`?

**Pull request merged**